docs/root/intro/version_history.rst

```diff
@@ -68,10 +68,10 @@ Version history
   :ref:`validated <envoy_api_field_route.RouteMatch.TlsContextMatchOptions.validated>`.
 * router: added support for :ref:`regex_rewrite
   <envoy_api_field_route.RouteAction.regex_rewrite>` for path rewriting using regular expressions and capture groups.
-* router: added support for DOWNSTREAM_LOCAL_PORT :ref:`header formatter <config_http_conn_man_headers_custom_request_headers>`.
+* router: added support for `%DOWNSTREAM_LOCAL_PORT%` :ref:`header formatter <config_http_conn_man_headers_custom_request_headers>`.
 * router: don't ignore :ref:`per_try_timeout <envoy_api_field_route.RetryPolicy.per_try_timeout>` when the :ref:`global route timeout <envoy_api_field_route.RouteAction.timeout>` is disabled.
 * router: strip whitespace for :ref:`retry_on <envoy_api_field_route.RetryPolicy.retry_on>`, :ref:`grpc-retry-on header <config_http_filters_router_x-envoy-retry-grpc-on>` and :ref:`retry-on header <config_http_filters_router_x-envoy-retry-on>`.
-* runtime: enabling the runtime feature "envoy.deprecated_features.allow_deprecated_extension_names"
+* runtime: enabling the runtime feature `envoy.deprecated_features.allow_deprecated_extension_names`
```

Comment:
period here too?

Also after flood_protection` above

Reply:
This wraps to the next line. I fixed the one above.

Reply:
Oh yeah, my bad.
I've also considered fixing multi-lines because they're more annoying to parse visually :-P